src/main/java/me/ryanhamshire/GriefPrevention/PlayerEventHandler.java

```diff
@@ -1564,7 +1565,7 @@ void onPlayerInteract(PlayerInteractEvent event)
                                 clickedBlockType == Material.LOOM ||
                                 clickedBlockType == Material.PUMPKIN ||
                                 clickedBlockType == Material.RESPAWN_ANCHOR ||
-                                clickedBlockType == Material.ROOTED_DIRT ||
+                                (clickedBlockType == Material.ROOTED_DIRT && player.getInventory().getItemInMainHand().getType().toString().toLowerCase().endsWith("_hoe")) ||
```

Comment:
Sorry, also completely blanked on this the first time looking at it - we should probably be using [`Tag.ITEMS_HOES`](https://hub.spigotmc.org/javadocs/spigot/org/bukkit/Tag.html#ITEMS_HOES) rather than checking the end of the name for better futureproofing/compatibility.

Reply:
How would I implement that? It was pretty late in my time so for whatever reason I was thinking it couldn't be used for this purpose

Reply:
`Tag.ITEMS_HOES` is a `Tag<Material>`, so you'd just use `Tag#isTagged` to check if another `Material` is in it. In this case it'd be the material of the item in the interacting hand, which is [available from the event](https://hub.spigotmc.org/javadocs/spigot/org/bukkit/event/player/PlayerInteractEvent.html#getMaterial()).

If you can't get there give me a shout and I can help, but I'll be pretty swamped until tomorrow night.

Reply:
Committing this now, could you please check my work?

Reply:
Almost - you nailed the tag usage, and the updated version would work in terms of preventing both hands successfully, but we can just check the material being interacted with to not have to do that. I sent you a PR on your fork that'll make the last change I had in mind.

Reply:
I didn't think to check if that was an option. Thank you :)